full-node/sov-ethereum/src/gas_price/gas_oracle.rs

```diff
@@ -303,6 +303,12 @@ fn effective_gas_tip(
     }
 }
 
+fn convert_u256_to_u64(u256: U256) -> Result<u64, TryFromSliceError> {
+    let bytes: [u8; 32] = u256.to_be_bytes();
+    let bytes: [u8; 8] = bytes[24..].try_into()?;
```

Comment:
Unrelated to this PR, but this looks infallible. If the array is `32` bytes, then it will always have `24 + 8` bytes, making this conversion check unnecessary.

Reply:
Fixed + added proptest